build/cloud/07-deployment.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: kubeedge
+    kubeedge: edgecontroller
+  name: edgecontroller
+  namespace: kubeedge
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kubeedge
+      kubeedge: edgecontroller
+  template:
+    metadata:
+      labels:
+        k8s-app: kubeedge
+        kubeedge: edgecontroller
+      annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
why is this annotation specified ?

Reply:
We can remove this, it's just used to disable istio sidecar injection

Reply:
better we remove. thanks.